jobs/job_test_utils.py

```diff
@@ -187,7 +187,7 @@ def assert_job_output_is_empty(self):
         self.assert_pcoll_empty(self.run_job())
 
 
-@contextlib2.contextmanager
+@contextlib.contextmanager
```

Comment:
Just to check... it is intentional to keep contextlib here?

Reply:
Yep that's right.